### PR TITLE
add features.md and fuel-core lib doc.

### PR DESCRIPTION
**Pull request by Salka1988**

### CODE REVIEW

File: docs/src/providers/features.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Features
+
+### Fuel-core lib
+
+The `fuel-core-lib` feature allows us to run a fuel-core node without installing the fuel-core binary on the local machine. This entails downloading all the dependencies needed to run the fuel-core node.
```
Comment (vnepveu) [inline docs/src/providers/features.md:5]: You should explain what dependencies it requires, maybe even put an example `Cargo.toml` file like in the `Setup` part.

**Review by vnepveu — CHANGES_REQUESTED** [review]
This is going in the right direction, but IMO you should be documenting how to use the feature (people unfamiliar would not necessarily know how to). Also I think you could put it inside the `Running a short-lived fuel-node` as a paragraph, I think it would make more sense, what do you think? Since it's used for connecting to a node in that case.

File: docs/src/providers/short-lived.md [hunk 2]
```diff
@@ -19,3 +19,13 @@ You can also use `launch_provider_and_get_wallet()`, which abstracts away the `s
 ```rust,ignore
 let wallet = launch_provider_and_get_wallet().await;
 ```
+
+# Features
+
+### Fuel-core lib
```
Comment (digorithm) [inline docs/src/providers/short-lived.md:25]:
```suggestion
## Fuel-core lib
```

File: docs/src/providers/short-lived.md [hunk 3]
```diff
@@ -19,3 +19,13 @@ You can also use `launch_provider_and_get_wallet()`, which abstracts away the `s
...
+
+The `fuel-core-lib` is a feature defined in `fuels` library, and it allows us to run a fuel-core node without installing the fuel-core binary on the local machine. This entails downloading all the dependencies needed to run the fuel-core node.
```
Comment (digorithm) [inline docs/src/providers/short-lived.md:27]: Clarity improvements:

```suggestion
The `fuel-core-lib` is a feature defined in the `fuels` library, allowing us to run a `fuel-core` node without installing the `fuel-core` binary on the local machine. Using the `fuel-core-lib` feature flag entails downloading all the dependencies needed to run the fuel-core node.
```

**Review by digorithm — CHANGES_REQUESTED** [review]

**Review by vnepveu — APPROVED** [review]
perfect! will be clearer for future users :1st_place_medal:

**Review by digorithm — APPROVED** [review]
:shipit:

**Pull request merged**